src/chips/nrf51822/gpio.rs

```diff
@@ -89,7 +196,33 @@ impl IndexMut<usize> for Port {
     }
 }
 
-pub static mut PA : Port = Port {
+impl Port {
+    // GPIOTE interrupt
+    pub fn handle_interrupt(&self) {
```

Comment:
I think this is relatively self-explanatory, but would be good to document what's going on. Basically, there is only one hardware interrupt for GPIOTE, so `handle_interrupt` looks at the interrupt status registers to see which pins fired, and invokes the corresponding pin's handle_interrupt function. Right?